app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/PlaceRegisterActivity.kt

```diff
@@ -41,12 +50,67 @@ class PlaceRegisterActivity : AppCompatActivity() {
 
         btn_address_search.setOnClickListener {
             val intent = Intent(this, SearchAddressActivity::class.java)
-            startActivity(intent)
+            startActivityForResult(intent, 1)
```

Comment:
```suggestion
            startActivityForResult(intent, SAMPLE_RQ_CODE)
```
request code는 의미를 담에서 상수로 관리하는것이 좋아보여요

Reply:
수정했습니당